### PR TITLE
👽️ Set default to `like` prefix

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: pages/in/settings/index.vue [hunk 1]
```diff
@@ -81,10 +81,10 @@
                     {{ $t('Register.form.walletAddress') }}:
                   </span>
                   <span
-                    v-if="getUserInfo.cosmosWallet"
+                    v-if="getUserInfo.likeWallet"
                     class="content"
                   >
-                    {{ getUserInfo.cosmosWallet }}
+                    {{ getUserInfo.likeWallet }}
```
Comment (nwingt) [inline pages/in/settings/index.vue:87]: This is conflict with https://github.com/likecoin/like-co/pull/1620

Comment (williamchong) [reply pages/in/settings/index.vue:87]: I have merged and rebase the feature branch, @WeiJun0827 please change accordingly thanks

Comment (WeiJun0827) [reply pages/in/settings/index.vue]: Rebased the branch, ready for review, thanks!

**Review by nwingt — APPROVED** [review]
I think it is more robust to add cosmosWallet as a fallback instead of replacing them all

**Review by williamchong — APPROVED** [review]

**Comment by williamchong** [comment]
We probably want to remove querying cosmosWallet from id/other user object in the future, and only keep the current user part in case user did not migrate

**Pull request merged**